custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py

```diff
@@ -100,8 +100,7 @@ def aggregate(self, cursor):
             cursor.execute(rollup_query)
 
         self.assert_no_awc_missing_from_new_table(cursor)
-        old_def, new_def = self.generate_diff_of_tables(cursor)
-        # TODO figure what to do with these
+        self.generate_diff_of_tables(cursor)
```

Comment:
decided to only send from the monolith task so that we aren't sending double emails